gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/mfa-activate.component.html

```diff
@@ -33,9 +33,9 @@ <h5>Activate MFA</h5>
     <form>
       <mat-form-field appearance="outline" floatLabel="always" fxFlex="40" style="margin-left: 5px;">
         <mat-select placeholder="Mode" name="mode" [(ngModel)]="this.currentMode" (selectionChange)="applyModeChange($event)">
-          <mat-option *ngFor="let mode of modes" [value]="mode" [gioLicense]="modeLicenseMetadata(mode)">
+          <mat-option *ngFor="let mode of modes" [value]="mode" [gioLicense]="mode.feature">
             {{ mode.label }}
-            <mat-icon *ngIf="mode.ee" svgIcon="gio:lock"></mat-icon>
+            <mat-icon *ngIf="mode?.isMissingFeature$ | async" svgIcon="gio:lock"></mat-icon>
```

Comment:
is the check `?` in the variable `mode` really necessary? if yes, then maybe it also needs to be added to the other uses of the variable?